### PR TITLE
Update: navbar filter-cat

### PR DESCRIPTION
**Pull request by harshitpaliwal95**
- update Navbar filter with category
- fix Reducer typo

### CODE REVIEW

**Review by vikrantkode — APPROVED** [review]
All Changes are looking good
Good to merge

File: src/components/filter/filter.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ export const Filter = () => {
               name="checkbox"
               type="radio"
               onChange={() =>
-                dispatch({ type: "sortBy", payload: "HIGH_TO_LOW_RATE" })
+                dispatch({ type: "SHORT_BY", payload: "HIGH_TO_LOW_RATE" })
               }
```
Comment (agrawalshanu708) [inline src/components/filter/filter.js:55]: change to SORT_BY

File: src/pages/homePage/home.js [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
+  const [product, setProduct] = useState([]);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await axios.get("/api/products");
+        setProduct(response.data.products);
+      } catch (error) {
+        console.log(error.message);
+      }
```
Comment (agrawalshanu708) [inline src/pages/homePage/home.js:18]: use console.error()

**Review by agrawalshanu708 — APPROVED** [review]
format your code by prettier
your functions are clean
good to merge

**Pull request merged**